### PR TITLE
enhance: help and feedback panel

### PR DESCRIPTION
**Pull request by jonathanyeung**
## enhance: help and feedback panel

Adds a Help and Feedback panel to the new Dendron View Container.

---

# Dendron Extended PR Checklist

- NOTE: the links don't work. you'll need to go into the wiki and use lookup to find the note until we fix some issues in the markdown export

### Basics

- [x] code should follow [Code Conventions](dev.process.code)
- [x] circular dependency check: make sure your code is not introducing new circular dependencies in plugin-core.  See [Avoiding Circular Dependencies](dev.process.code.best-practices).
- [x] sticking to existing conventions instead of creating new ones
  - eg: [if configuration for utilities are already in one module or package, add future utilities there as well](https://github.com/dendronhq/dendron/pull/1960#discussion_r786228021)

### Extended
- General
  - [x] check whether code be simplified
  - [x] check if similar function already exist in the codebase. if so, can it be re-used?
  - [x] check if this change adversely impact performance
- Operations
  - [x] when shipping this change, will it just work or will it introduce additional operational overhead due to complicated interface or known bugs?
- Architecture
  - [x] check if code is introducing changes on a foundational class or interface. if so, call for design review if needed
    - eg: [making changes to DNode](https://github.com/dendronhq/dendron/pull/2158#pullrequestreview-854689586)

### Basics

Added to Airtable / Upcoming
- [x] if you are adding analytics related changes, make sure the [Telemetry](https://wiki.dendron.so/notes/84df871b-9442-42fd-b4c3-0024e35b5f3c.html) docs are updated

### Extended
- [ ] can we track the performance of this change to know if it is _successful_?
  - eg: [see usage for export pod](https://github.com/dendronhq/dendron/pull/2190#pullrequestreview-855715612)

### Basics

- [N/A] [Write Tests](dev.process.qa.test) 
- [x] [Confirm existing tests pass](dev.process.qa.test)
- [x] [Confirm manual testing](dev.process.qa.test) 
- [x] Common cases tested
- [x] 1-2 Edge cases tested
- [x] If your tests changes an existing snapshot, snapshots have been [updated](dev.process.qa.test)

### CODE REVIEW

File: packages/plugin-core/src/features/HelpFeedbackTreeview.ts [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+      case MenuItem.reviewIssues:
+        openUrl("https://github.com/dendronhq/dendron/issues");
+        break;
+
+      case MenuItem.reportIssue:
+        openUrl("https://github.com/dendronhq/dendron/issues/new/choose");
+        break;
+
+      case MenuItem.joinCommunity:
+        openUrl("https://discord.gg/S5MQrwZD7R");
```
Comment (jonathanyeung) [inline packages/plugin-core/src/features/HelpFeedbackTreeview.ts:118]: @kevinslin - can you get me a Discord link? Right now, it'll say "invited by mobius" - do we have a general account or something like that?

File: packages/plugin-core/src/features/HelpFeedbackTreeview.ts [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+    switch (item) {
+      case MenuItem.getStarted:
+        openUrl(
+          "https://wiki.dendron.so/notes/678c77d9-ef2c-4537-97b5-64556d6337f1/"
+        );
+        break;
+
+      case MenuItem.readDocs:
+        openUrl("https://wiki.dendron.so/notes/F5ly88oU0ecUiACLxET5E/");
```
Comment (jonathanyeung) [inline packages/plugin-core/src/features/HelpFeedbackTreeview.ts:106]: This links to the 'Basics' page - open to suggestions on a better landing zone (or if we think this item is redundant with the 'Getting Started' link above, which also links to wiki.dendron.so

**Comment by jonathanyeung** [comment]
image

File: packages/plugin-core/src/features/HelpFeedbackTreeview.ts (hunk 1 above)
Comment (kevinslin) [reply packages/plugin-core/src/features/HelpFeedbackTreeview.ts:118]: see https://link.dendron.so/discord

also, frequently used links are kept in [[Common Links|dendron://dendron.handbook/area.community.ref.links]]

File: packages/plugin-core/src/features/HelpFeedbackTreeview.ts (hunk 2 above)
Comment (kevinslin) [reply packages/plugin-core/src/features/HelpFeedbackTreeview.ts:106]: would use user guide -> https://wiki.dendron.so/notes/FWtrGfE4YJc3j0yMNjBn5/

File: packages/plugin-core/src/features/HelpFeedbackTreeview.ts [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+        break;
+
+      case MenuItem.reviewIssues:
+        openUrl("https://github.com/dendronhq/dendron/issues");
```
Comment (kevinslin) [inline packages/plugin-core/src/features/HelpFeedbackTreeview.ts:110]: will people actually click this? we can leave this in here to test

**Review by kevinslin — APPROVED** [review]

**Comment by kevinslin** [comment]
looks good, feel free to merge after you've updated the links

File: packages/plugin-core/src/features/HelpFeedbackTreeview.ts (hunk 3 above)
Comment (jonathanyeung) [reply packages/plugin-core/src/features/HelpFeedbackTreeview.ts:110]: Not sure - but I'm following the pattern set by both `Project Manager` and `Remote SSH`.  I think the idea is that they want people to look at existing issues before filing a new one.

**Pull request merged**